reference/forms/types/money.rst

```diff
@@ -16,6 +16,7 @@ how the input and output of the data is handled.
 +-------------+---------------------------------------------------------------------+
 | Options     | - `currency`_                                                       |
 |             | - `divisor`_                                                        |
+|             | - `rounding_mode`_                                                  |
```

Comment:
should be after `grouping` IMO to sort the options alphabetically

Reply:
@xabbuh done